app/src/shared/test/modules/resourceKey/services.test.js

```diff
@@ -0,0 +1,170 @@
+            assert.deepStrictEqual(result, data)
+        })
+    })
+
+    describe('getStreamResourceKeys', () => {
+        it('gets API keys fro stream', async () => {
+            const streamId = '1234'
+            const data = [
+                {
+                    id: 'zZr2TzwuRriiSAYAoY2IzwlbFPzNc0RmyJAGMbInmCIw',
+                    name: 'Test',
+                    user: 'tester1@streamr.com',
+                },
+            ]
+
+            moxios.wait(() => {
+                const request = moxios.requests.mostRecent()
+                request.respondWith({
+                    status: 200,
+                    response: data,
+                })
+
+                assert.equal(request.config.method, 'get')
+                assert.equal(request.config.url, `/streams/${streamId}/keys`)
+            })
+
+            const result = await services.getStreamResourceKeys(streamId)
+            assert.deepStrictEqual(result, data)
+        })
+    })
+
+    describe('addMyResourceKey', () => {
+        it('makes a POST request', async () => {
+            const name = 'new key'
+            const data = {
+                id: 'zZr2TzwuRriiSAYAoY2IzwlbFPzNc0RmyJAGMbInmCIw',
+                name,
+                user: 'tester1@streamr.com',
+            }
+
+            moxios.wait(() => {
+                const request = moxios.requests.mostRecent()
+                request.respondWith({
+                    status: 200,
+                    response: data,
+                })
+
+                assert.equal(request.config.method, 'post')
+                assert.equal(request.config.url, '/users/me/keys')
+            })
+
+            const result = await services.addMyResourceKey(name)
+            assert.deepStrictEqual(result, data)
+        })
+    })
+
+    describe('addStreamResourceKey', () => {
+        it('makes a POST request', async () => {
+            const streamId = '1234'
+            const name = 'new key'
+            const data = {
+                id: 'zZr2TzwuRriiSAYAoY2IzwlbFPzNc0RmyJAGMbInmCIw',
+                name,
+                user: 'tester1@streamr.com',
+            }
+
+            moxios.wait(() => {
+                const request = moxios.requests.mostRecent()
+                request.respondWith({
+                    status: 200,
+                    response: data,
+                })
+
+                assert.equal(request.config.method, 'post')
+                assert.equal(request.config.url, `/streams/${streamId}/keys`)
+            })
+
+            const result = await services.addStreamResourceKey(streamId, name)
+            assert.deepStrictEqual(result, data)
+        })
+    })
+
+    describe('removeMyResourceKey', () => {
+        it('makes a DELETE request', async () => {
+            const id = 'testid'
+            moxios.wait(() => {
+                const request = moxios.requests.mostRecent()
+                request.respondWith({
+                    status: 201,
```

Comment:
I suppose it doesn't matter too much but I think this should be `status: 204` rather than `status: 201`. At least according to the API docs, though both pass the test.  

201 Created
The request has been fulfilled, resulting in the creation of a new resource.

204 No Content
The server successfully processed the request and is not returning any content.